pkg/model/awsmodel/autoscalinggroup.go

```diff
@@ -542,5 +542,11 @@ func (b *AutoscalingGroupModelBuilder) buildAutoScalingGroupTask(c *fi.ModelBuil
 		t.MixedSpotMaxPrice = ig.Spec.MaxPrice
 	}
 
+	if ig.Spec.MaxInstanceLifetime != nil {
+		lifetimeSec := int64(ig.Spec.MaxInstanceLifetime.Seconds())
+		t.MaxInstanceLifetime = fi.Int64(lifetimeSec)
+	} else {
+		t.MaxInstanceLifetime = fi.Int64(0)
```

Comment:
the e2e test is failing with:

`Error: error running tasks: deadline exceeded executing task AutoscalingGroup/master-eu-west-1a.masters.e2e-pr13892.pull-kops-e2e-kubernetes-aws.test-cncf-aws.k8s.io. Example error: error creating AutoScalingGroup: maxInstanceLifetime must be between 86400 and 31536000 seconds (inclusive).`

```suggestion
```

and then `./hack/update-expected.sh` should remove all of the `max_instance_lifetime = 0` below

Reply:
Thx, fixed.